apps/condo/lang/en.json

```diff
@@ -236,6 +236,20 @@
   "pages.condo.ticket.title.ClientInfo": "Client information",
   "pages.condo.ticket.title.TicketInfo": "Task information",
   "pages.condo.ticket.title.UserInfo": "Resident information",
+  "pages.condo.ticket.title.TicketChanges": "Change log of the ticket",
```

Comment:
line number inside en != line number inside ru

Reply:
Fixed in pull-request #130